### PR TITLE
Authentication section for the new admin UI

### LINKED ISSUE #1488: Configuring authentication chapter
### Description

Update this Server Administration Guide chapter for the new Admin Console

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by edewit**
fixes: #1488

### CODE REVIEW

File: server_admin/topics.adoc [hunk 1]
```diff
@@ -26,10 +26,20 @@ include::topics/assembly-roles-groups.adoc[]
 include::topics/authentication.adoc[]
 include::topics/authentication/password-policies.adoc[]
 include::topics/authentication/otp-policies.adoc[]
+ifeval::[{project_community}==true]
 include::topics/authentication/flows.adoc[]
+endif::[]
+ifeval::[{project_product}==true]
+include::events/flows-prod.adoc[]
```
Comment (andymunro) [inline server_admin/topics.adoc]: I think this file should be here:  topics/authentication/flows-prod.adoc

File: server_admin/topics.adoc [hunk 2]
```diff
@@ -26,10 +26,20 @@ include::topics/assembly-roles-groups.adoc[]
...
+endif::[]
 include::topics/authentication/kerberos.adoc[]
 include::topics/authentication/x509.adoc[]
+ifeval::[{project_community}==true]
 include::topics/authentication/webauthn.adoc[]
+endif::[]
+ifeval::[{project_product}==true]
```
Comment (andymunro) [inline server_admin/topics.adoc:40]: should this be webauthn-prod.adoc

File: server_admin/topics/authentication/webauthn.adoc [hunk 3]
```diff
@@ -17,38 +17,33 @@ The setup procedure of WebAuthn support for 2FA is the following :
...
+. Toggle the *Webauthn Register* switch to *ON*.
 
-Mark the *Default Action* checkbox if you want all new users to be required to register their WebAuthn credentials.
+Toggle the *Default Action* switch to *ON* if you want all new users to be required to register their WebAuthn credentials.
 
 [[_webauthn-authenticator-setup]]
 ===== Adding WebAuthn authentication to a browser flow
 
 . Click *Authentication* in the menu.
 . Click the *Browser* flow.
```
Comment (andymunro) [inline server_admin/topics/authentication/webauthn.adoc:28]: If you click the Browser flow, it opens up a different dialog. Keep this step, but then modify the next step as indicated.

File: server_admin/topics/authentication/webauthn.adoc [hunk 4]
```diff
@@ -17,38 +17,33 @@ The setup procedure of WebAuthn support for 2FA is the following :
...
-. Click *Copy* to make a copy of the built-in *Browser* flow.
+. Select *Duplicate* from the "ellipses button" to make a copy of the built-in *Browser* flow.
```
Comment (andymunro) [inline server_admin/topics/authentication/webauthn.adoc]: From the *Action* list, select *Duplicate*.

For more details on wording for GUI elements, see https://docs.engineering.redhat.com/display/CCSENABLE/GUI+elements

File: server_admin/topics/authentication/webauthn.adoc [hunk 5]
```diff
@@ -17,38 +17,33 @@ The setup procedure of WebAuthn support for 2FA is the following :
...
 . Enter a name for the copy.
-. Click *Ok*.
-. Click the _Actions_ link for *WebAuthn Browser Browser - Conditional OTP* and click _Delete_.
-. Click *Delete*.
+. Click *Duplicate*.
+. Click on the *🗑️* (trash can icon) of the "WebAuthn Browser Browser - Conditional OTP" and click *Delete*.
```
Comment (andymunro) [inline server_admin/topics/authentication/webauthn.adoc]: I don't see "WebAuthn Browser Browser - Conditional OTP" even if I do it the way you describe with the ellipses.
Can you verify if something is missing.

**Comment by andymunro** [comment]
@edewit I will need to finish reviewing on Monday. It looks good overall.  However, I don't see the password-policies file in the list of changed files.

File: server_admin/topics/authentication/webauthn.adoc (hunk 3 above)
Comment (edewit) [reply server_admin/topics/authentication/webauthn.adoc:28]: No it doesn't open a dialog it shows the details of the browser flow

**Comment by edewit** [comment]
I didn't have to change the password-policies file as it works exactly
 the same as in the old console it only looks different, so I only updated the images.

**Comment by andymunro** [comment]
> I didn't have to change the password-policies file as it works exactly the same as in the old console it only looks different, so I only updated the images.

The difference is that in the new console, you click Policies and then the Password Policies tab appears.

File: server_admin/topics/authentication/flows.adoc [hunk 6]
```diff
@@ -7,18 +7,17 @@ To view all the flows, actions, and checks, each flow requires:
 
 .Procedure
 . Click *Authentication* in the menu.
```
Comment (andymunro) [inline server_admin/topics/authentication/flows.adoc]: A single-step procedure looks weird.  Please move it down to just before line 15, "Click on the _Browser_

File: server_admin/topics/authentication/flows.adoc [hunk 7]
```diff
@@ -7,18 +7,17 @@ To view all the flows, actions, and checks, each flow requires:
...
-. Click the *Flows* tab.
 
 ==== Built-in flows
 
 {project_name} has several built-in flows. You cannot modify these flows, but you can alter the flow's requirements to suit your needs.
 
-In the drop-down list, select *browser* to display the Browser Flow screen.
+Click on the _Browser_ item in the list to see the details.
```
Comment (andymunro) [inline server_admin/topics/authentication/flows.adoc]: Make this step two of the procedure.

**Comment by edewit** [comment]
> > I didn't have to change the password-policies file as it works exactly the same as in the old console it only looks different, so I only updated the images.
> 
> The difference is that in the new console, you click Policies and then the Password Policies tab appears.

Ahh yes, I see, changed it

File: server_admin/topics/authentication/password-policies.adoc [hunk 8]
```diff
@@ -7,7 +7,12 @@ When {project_name} creates a realm, it does not associate password policies wit
...
 .Procedure
 . Click *Authentication* in the menu.
+ifeval::[{project_community}==true]
+. Click the *Policies* tab.
+endif::[]
+ifeval::[{project_product}==true]
 . Click the *Password Policy* tab.
+endif::[]
 . Select the policy to add in the *Add policy* drop-down box.
 . Enter a value for the *Policy Value* corresponding with the policy chosen.
```
Comment (andymunro) [inline server_admin/topics/authentication/password-policies.adoc:21]: There is no *Policy Value* in the new UI.  Just a place where a value show up.  I think you can just eliminate the *Policy Value* phrase to work for both old and new UI.

. Enter a value that applies to the policy chosen.

File: server_admin/topics/authentication/webauthn.adoc [hunk 9]
```diff
@@ -17,38 +17,34 @@ The setup procedure of WebAuthn support for 2FA is the following :
...
-. Click *Copy* to make a copy of the built-in *Browser* flow.
-. Enter a name for the copy.
-. Click *Ok*.
-. Click the _Actions_ link for *WebAuthn Browser Browser - Conditional OTP* and click _Delete_.
-. Click *Delete*.
+. Select *Duplicate* from the "Action list" to make a copy of the built-in *Browser* flow.
+. Enter "WebAuthn Browser" as the name of the copy.
+. Click *Duplicate*.
+. Click on the name to go to the details
+. Click on the *🗑️* (trash can icon) of the "WebAuthn Browser Browser - Conditional OTP" and click *Delete*.
```
Comment (andymunro) [inline server_admin/topics/authentication/webauthn.adoc]: We always say "click" not "click on"

File: server_admin/topics/authentication/webauthn.adoc [hunk 10]
```diff
@@ -59,37 +55,31 @@ If a user does not have WebAuthn credentials, the user must register WebAuthn cr
...
-. Click on the _Actions_ link for *Conditional 2FA*.
-. Click *Add execution*.
-. Click the *Provider* drop-down list.
-. Click *WebAuthn Authenticator*.
-. Click *Save*.
-. Click _ALTERNATIVE_ for *Conditional 2FA*
+. Click *+* menu of the *WebAuthn Browser Forms* row.
+. Click *Add sub-flow*.
+. Enter "Conditional 2FA" for the _name_ field.
+. Click *Add*.
```
Comment (andymunro) [inline server_admin/topics/authentication/webauthn.adoc]: On the *WebAuthn Browser Forms* row, click the plus sign *+* and select *Add step*.

File: server_admin/topics/authentication/webauthn.adoc [hunk 11]
```diff
@@ -59,37 +55,31 @@ If a user does not have WebAuthn credentials, the user must register WebAuthn cr
...
+. Select *Conditional* for the *Conditional 2FA* to set its requirement to conditional.
+. Click *+* menu of the *Conditional 2FA* row.
+. Click *Add condition*.
```
Comment (andymunro) [inline server_admin/topics/authentication/webauthn.adoc]: On the *Conditional 2FA* row, click the plus sign *+* and select *Add condition*.

File: server_admin/topics/authentication/webauthn.adoc [hunk 12]
```diff
@@ -59,37 +55,31 @@ If a user does not have WebAuthn credentials, the user must register WebAuthn cr
...
 
 .Procedure
-. Click the _Actions_ link for *Conditional 2FA*.
-. Click *Add execution*.
-. Click the *Provider* drop-down list.
-. Click *ITP Form*.
-. Click *Save*.
-. Click _ALTERNATIVE_ for *Conditional 2FA*
+. Click *+* menu of the *Conditional 2FA* row.
+. Click *Add step*.
```
Comment (andymunro) [inline server_admin/topics/authentication/webauthn.adoc]: On the  *Conditional 2FA* row, click the plus sign *+* and select *Add step*.

File: server_admin/topics/authentication/x509.adoc [hunk 13]
```diff
@@ -252,6 +277,20 @@ If set, the client certificate trust chain will be always verified at the applic
...
 
+
+. Click *Authentication* in the menu.
+ifeval::[{project_community}==true]
+. Select *Duplicate* from the "Action list" to make a copy of the built-in "Direct grant" flow.
+. Enter a name for the copy.
+. Click *Duplicate*.
+. Click the created flow.
+. Click on the *🗑️* (trash can icon) of the "Username Validation" and click *Delete*.
+. Click on the *🗑️* (trash can icon) of the "Password" and click *Delete*.
```
Comment (andymunro) [inline server_admin/topics/authentication/x509.adoc]: Click the trash can icon 🗑️ of the,,,,

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good.  It made suggestions based on the official style guidelines.  It looks like an icon included is okay, such as CogWheel, if that name is not clear: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui
I will still need to check a couple of items with you.  I will try to connect earlier on Friday.

File: server_admin/topics/authentication/x509.adoc (hunk 13 above)
Comment (edewit) [reply server_admin/topics/authentication/x509.adoc:288]: Guess you mean Click _on_ the trash can icon

File: server_admin/topics/authentication/x509.adoc [hunk 14]
```diff
@@ -163,6 +163,16 @@ endif::[]
 [[_browser_flow]]
 ==== Adding X.509 client certificate authentication to browser flows
 
+ifeval::[{project_community}==true]
+. Click *Authentication* in the menu.
+. Select *Duplicate* from the "Action list" to make a copy of the built-in "Browser" flow.
```
Comment (andymunro) [inline server_admin/topics/authentication/x509.adoc]:
```suggestion
. Click the *Browser* flow.
. From the *Action* list, select *Duplicate*. 
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks great, @edewit.  A few minor changes only.

**Review by andymunro — DISMISSED** [review]
Looks good. Thanks, @edewit

**Comment by andymunro** [comment]
Thanks, @edewit. Looks great.  Another one knocked off and ready to be merged, @ssilvert

**Review by andymunro — APPROVED** [review]

**Review by ssilvert — APPROVED** [review]

**Pull request merged**